src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -431,8 +443,18 @@ def deploy_arm_template_at_resource_group(cmd,
                                           resource_group_name=None,
                                           template_file=None, template_uri=None, parameters=None,
                                           deployment_name=None, mode=None, rollback_on_error=None,
-                                          no_wait=False, handle_extended_json_format=None,
-                                          aux_subscriptions=None, aux_tenants=None, no_prompt=False):
+                                          no_wait=False, aux_subscriptions=None, aux_tenants=None,
+                                          confirm_with_what_if=None, what_if_result_format=None,
+                                          no_prompt=False):
```

Comment:
Same as above

Reply:
Good catch. I think it was deleted by accident after resolving some rebasing conflicts. I added it back.